.travis.yml

```
@@ -35,6 +35,9 @@ env:
 matrix:
   fast_finish: true
   include:
+    - compiler: gcc
+      env: HOST=s390x-unknown-linux-gnu DOCKER_NAME_TAG=s390x/ubuntu:18.04
```

What does `DOCKER_NAME_TAG` do?
```suggestion
      env: HOST=s390x-unknown-linux-gnu
```

---

I have no idea anymore, I assume I copied it from somewhere.